<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="2" total="3" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="97161" started-at="2018-09-28T11:58:22Z" finished-at="2018-09-28T12:00:00Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="97161" started-at="2018-09-28T11:58:22Z" finished-at="2018-09-28T12:00:00Z">
      <class name="WordpressQA.ForgotPassword">
        <test-method status="PASS" signature="setup()[pri:0, instance:WordpressQA.ForgotPassword@1bce4f0a]" name="setup" is-config="true" duration-ms="11935" started-at="2018-09-28T11:58:22Z" finished-at="2018-09-28T11:58:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="ForgotPwd()[pri:2, instance:WordpressQA.ForgotPassword@1bce4f0a]" name="ForgotPwd" duration-ms="85154" started-at="2018-09-28T11:58:34Z" finished-at="2018-09-28T11:59:59Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at WordpressQA.ForgotPassword.ForgotPwd(ForgotPassword.java:73)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ForgotPwd -->
        <test-method status="PASS" signature="teardown(org.testng.ITestResult)[pri:0, instance:WordpressQA.ForgotPassword@1bce4f0a]" name="teardown" is-config="true" duration-ms="39" started-at="2018-09-28T11:59:59Z" finished-at="2018-09-28T12:00:00Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=ForgotPwd status=FAILURE method=ForgotPassword.ForgotPwd()[pri:2, instance:WordpressQA.ForgotPassword@1bce4f0a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- WordpressQA.ForgotPassword -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
